tlm_teamd/values_store.cpp

```diff
@@ -366,7 +366,13 @@ void ValuesStore::update(const std::vector<StringPair> & dumps)
     {
         const auto & storage = from_json(dumps);
         const auto & old_keys = get_old_keys(storage);
-        remove_keys_db(old_keys);
+        // Do not delete te key from State Db. State DB LAB_TABLE entry is created/deleted
+        // from teamsyncd on detecting netlink of teamd dev as up/down. For some reason 
+        // if we do not get state dump from teamdctl it might be transient issue. If it is 
+        // persistent issue then teamsyncd might be able to catch it and delete state db entry
+        // or we can keep entry in it's current state as best effort. Similar to try_add_lag which is best effort
+        // to connect to teamdctl and if it fails we do not delete State Db entry.
+        //remove_keys_db(old_keys);
         remove_keys_storage(old_keys);
         const auto & keys_to_refresh = update_storage(storage);
         update_db(storage, keys_to_refresh);
```

Comment:
This is a good find ! 
Do we need to update_db() here. The idea of tlm_teamd I remember was just to get the data from db.

Reply:
no @judyjoseph . tlm_tead gets data via `teamdctl` and put into STATE_DB which is used as part of `show commands`

Eg:

This is the state db with tlm_teamd:
  "LAG_TABLE|PortChannel01": {
    "expireat": 1729198772.0626316,
    "ttl": -0.001,
    "type": "hash",
    "value": {
      "admin_status": "up",
      "mtu": "9100",
      "oper_status": "down",
      "runner.active": "true",
      "runner.fallback": "false",
      "runner.fast_rate": "false",
      "setup.kernel_team_mode_name": "loadbalance",
      "setup.pid": "26",
      "state": "ok",
      "team_device.ifinfo.dev_addr": "68:79:09:2d:68:c2",
      "team_device.ifinfo.ifindex": "200"
    }